Nonprofit-Exchange-Hub/web-app
# Org sign up w/ image part 1a #208

basic pieces of https://github.com/Nonprofit-Exchange-Hub/web-app/issues/208
does not include file size validation
does not include writing to AWS

adds image_url property to FE signup form data and validators, BE entity and req body validators




## Test Plan


repro steps:

1. http://localhost:3000/signup-nonprofit
2. fill out entire form including org profile image input (just use a url for now)
3. submit
4. see in server logs or via postgres `select id, name, ein, image_url from organizations order by id DESC limit 1;` that data, img made it to BE 

## Resources

- [opted not to allow heic uploads](https://itnext.io/tackling-iphone-or-ipad-images-support-in-browser-8e3e64e9aaa1)
